File: Lv1/제일_작은_수_제거하기/Solution.js
function solution(arr) {
    // 배열의 길이가 1이면 [-1] 반환
    if (arr.length === 1) return [-1];

    // 가장 작은 값 찾기
    let min = Math.min(...arr);

    // 가장 작은 값을 제외한 배열 생성
    return arr.filter(num => num !== min);
}

File: Lv1/제일_작은_수_제거하기/Solution.java
package Lv1.제일_작은_수_제거하기;

import java.util.ArrayList;
import java.util.Arrays;

class Solution {
    public int[] solution(int[] arr) {
        int[] answer = {};
    if(arr.length == 1){
        return new int[]{-1};
    }

    int min = Arrays.stream(arr).min().getAsInt();

    ArrayList<Integer> list = new ArrayList<>();
    for(int num : arr){
        if(num != min){
            list.add(num);
        }
    }

    answer = list.stream().mapToInt(i -> i).toArray();
        return answer;
    }
}